apptentive-feedback/src/main/java/apptentive/com/android/feedback/conversation/ConversationManager.kt

```diff
@@ -507,7 +514,7 @@ internal class ConversationManager(
         val conversation = activeConversationSubject.value
         val configuration = conversation.configuration
 
-        if (isInThePast(configuration.expiry) || isDebuggable) {
+        if (isInThePast(configuration.expiry) || isDebuggable || !DependencyProvider.of<AndroidSharedPrefDataStore>().containsKey(SDK_CORE_INFO, METRICS_ENABLED)) {
```

Comment:
I had to add this check to ensure the Configuration is fetched if the metrics value is not found the shared pref which means the app is launched for the first time after enabling the metrics value.